bindings_ffi/gen_kotlin.sh

```diff
@@ -24,7 +24,7 @@ cargo run --bin ffi-uniffi-bindgen \
   --manifest-path $BINDINGS_MANIFEST \
   --features uniffi/cli --release -- \
   generate \
-  --lib-file $TARGET_DIR/release/lib$PROJECT_NAME.dylib $BINDINGS_PATH/src/$PROJECT_NAME.udl \
+  --lib-file $TARGET_DIR/release/lib$PROJECT_NAME.dylib \
```

Comment:
I think this is causing errors now for me. Can you run this script and see? I'm getting `error: the following required arguments were not provided:
  <SOURCE>`